doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-convert-proc.adoc

```diff
@@ -11,7 +11,7 @@ ifdef::DROOLS,JBPM,OP[]
 xref:decision-tables-con_decision-tables[].
 endif::[]
 
-WARNING: Guided decision tables and spreadsheet decision tables are different decision table formats that support different features. Any supported features that differ between the two decision table formats are modified or lost when you convert one decision table format to the other.
+WARNING: Guided decision tables and spreadsheet decision tables are different decision table formats that support different features. Any supported features that differ between the two decision table formats (Hit policy, for example) are modified or lost when you convert one decision table format to the other.
```

Comment:
Looks good. Minor comments.

A lot of instances of "decision". I think that one can be removed. Also, I suggest the change to the example.

WARNING: Guided decision tables and spreadsheet decision tables are different table formats that support different features. Supported features that differ between the two decision table formats (for example, Hit policy) are modified or lost when you convert one decision table format to the other.

Reply:
Done